PyInstaller/utils/win32/icon.py

```diff
@@ -145,20 +145,23 @@ def CopyIcons_FromIco(dstpath, srcpath, id=1):
     icons = map(IconFile, srcpath)
     logger.info("Copying icons from %s", srcpath)
 
-    hdst = win32api.BeginUpdateResource(dstpath, 0)
-
-    iconid = 1
-    # Each step in the following enumerate() will instantiate an IconFile
-    # object, as a result of deferred execution of the map() above.
-    for i, f in enumerate(icons):
-        data = f.grp_icon_dir()
-        data = data + f.grp_icondir_entries(iconid)
-        win32api.UpdateResource(hdst, RT_GROUP_ICON, i, data)
-        logger.info("Writing RT_GROUP_ICON %d resource with %d bytes", i, len(data))
-        for data in f.images:
-            win32api.UpdateResource(hdst, RT_ICON, iconid, data)
-            logger.info("Writing RT_ICON %d resource with %d bytes", iconid, len(data))
-            iconid = iconid + 1
+    hdst = win32api.BeginUpdateResource(dstpath, 1)
```

Comment:
Per https://pywin32-ctypes.readthedocs.io/en/stable/api/win32ctypes.pywin32.win32api.html#win32ctypes.pywin32.win32api.BeginUpdateResource, the second parameter should be True, not 1. Add a comment along the lines of "Remove any existing resources from the file." Is this OK -- are there any resources embedded in the file we don't want to delete?

Reply:
It was 0 before, so I figured, a 1 would be acceptable now, but OK.
Icon handling is the first thing we do with the resources, so deleting everything at this point should be fine, I checked.

Reply:
@bjones1 no it should be fine.